string departure_station
string arrival_destination
string date_of_journey
string departure_time
string arrival_time
string actual_arrival_time  # Può essere vuoto in caso di cancellazione
string journey_status
string reason_for_delay #Può essere vuoto se non c'è stato ritardo